pkg/autohealing/controller/controller.go

```diff
@@ -65,6 +65,11 @@ const (
 	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
 )
 
+var (
+	masterUnhealthyNodes []healthcheck.NodeInfo
+	workerUnhealthyNodes []healthcheck.NodeInfo
+)
+
```

Comment:
Do we really need those two vars here given they are defined in L392?

Reply:
They're global variables, called at line 348 and 372.

Reply:
Right, they are defined here but re-initialized in each loop, I got it.